katello_certs/hooks/boot/02-message-helpers.rb

```diff
@@ -40,17 +40,13 @@ def proxy_instructions_message(kafo)
 
   To finish the installation, follow these steps:
 
-  If you do not have the #{proxy_name} registered to the #{main_instance_name} instance, then please do the following:
-
-  1. yum -y localinstall http://#{fqdn}/pub/katello-ca-consumer-latest.noarch.rpm
-  2. subscription-manager register --org "#{HighLine.color(org, :info)}"
-
   Once this is completed run the steps below to start the #{proxy_name} installation:
 
-  1. Ensure that the #{installer_package} package is installed on the system.
-  2. Copy the following file <%= color("#{certs_tar}", :info) %> to the system <%= color("#{foreman_proxy_fqdn}", :info) %> at the following location <%= color("#{certs_tar_file}", :info) %>
+  1. Register the #{proxy_name} to the #{main_instance_name} instance using global registration.
```

Comment:
```suggestion
  1. Register the #{proxy_name} to the #{main_instance_name} instance.
```

I'd not even mention "global registration" by name here